docs/modules/ROOT/pages/how-to-guides/proc_release_plan.adoc

```diff
@@ -0,0 +1,59 @@
+= Creating a `ReleasePlan` object
+
+The development team creates a `ReleasePlan` object in the developer workspace. The `ReleasePlan` object includes a reference to the application that the development team wants to release, along with the namespace and workspace where the application is supposed to be released.
```

Comment:
Isn't `namespace` and `workspace` the same thing now that we don't have KCP?

Reply:
Removed namespace.